index.js

```diff
@@ -18,7 +17,7 @@ const parsersKeys = Object.keys(parsers);
 // but this is too generic to be added here, so we playing safe and not applying it.
 function isBadgeUrl(url) {
     return ['.svg', '.png', '.jpg', '.gif', 'svg=true', 'png=true', 'badge', 'appveyor']
-    .some(extension => url.indexOf(extension) !== -1);
+        .some(extension => url.indexOf(extension) !== -1);
```

Comment:
Please don't do unrelated changes. This is not respecting the project's eslint configuration. Do you have your editor setup correctly?

Reply:
There might be a problem with the configuration. `npm run lint` complains about the indentation if I changed it back.